src/Panel.js

```diff
@@ -23,6 +25,18 @@ const Panel = React.createClass({
     };
   },
 
+  getInitialState(){
+    let defaultExpanded = this.props.defaultExpanded != null ?
+      this.props.defaultExpanded :
+        this.props.expanded != null ?
+        this.props.expanded :
```

Comment:
This is still too high for me :smile: 
Just trying to ask the right questions :wink: